### ISSUE TITLE
resolve failing checks with tidyr CRAN upgrade

### ISSUE THREAD

**Issue by corybrunson** [corybrunson/ggalluvial#34]
This is from at automated email about [v1.0.0](https://github.com/tidyverse/tidyr/blob/master/NEWS.md), scheduled for CRAN on 2019-09-09:

```
 ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Did you misspecify an argument?
  1: suppressWarnings(tidyr::gather(dplyr::mutate(titanic_alluvia,
Index = 1:nrow(titanic_alluvia)), "Variable", "Value",
         axes = 1:4, factor_key = TRUE)) at testthat/test-is-alluvial.r:36
  2: withCallingHandlers(expr, warning = function(w)
invokeRestart("muffleWarning"))
  3: tidyr::gather(dplyr::mutate(titanic_alluvia, Index =
1:nrow(titanic_alluvia)), "Variable", "Value", axes = 1:4, factor_key
= TRUE)
  4: ellipsis::check_dots_unnamed() at
/private/tmp/RtmpBEzAiw/R.INSTALL570366266439/tidyr/R/gather.R:95
  5: stop_dots(message = paste0(length(named), " components of
`...` had unexpected names."), dot_names = named, .subclass =
"rlib_error_dots_named")
  6: abort(message, .subclass = c(.subclass, "rlib_error_dots"), ...)

  ══ testthat results
═════════════════════════════════════════════════════════════════════════════════════════════
  [ OK: 12 | SKIPPED: 0 | WARNINGS: 0 | FAILED: 1 ]
  1. Error: (unknown) (@test-is-alluvial.r#36)

  Error: testthat unit tests failed
  Execution halted
```

**Comment by corybrunson** [comment]
This seems to have been due to injudicious manual conversions between alluvia and lodes form in the test script, rather than to fundamental changes in tidyr. After some fixes in 6daec653e0e48c31582f21128b239e274128f383, the package passes the tests with the current devel version of tidyr installed. Plan to resubmit to CRAN on 2019-09-09 and watch out for errors immediately after.

**Comment by corybrunson** [comment]
v0.10.0 is on CRAN. Leaving this issue open until tidyr v1.0.0 is published.
